src/core/core.tooltip.js

```diff
@@ -839,13 +839,14 @@ module.exports = function(Chart) {
 			var me = this;
 			var options = me._options;
 			var changed = false;
+			var border = me._chart.chartArea;
 
 			me._lastActive = me._lastActive || [];
 
 			// Find Active Elements for tooltips
 			if (e.type === 'mouseout') {
 				me._active = [];
-			} else {
+			} else if (e.x > border.left && e.x < border.right) {
```

Comment:
I think limits should be included because values can be positioned at the start and end pixels.

Shouldn't we do the same for `e.y` but also clear active elements if mouse outside?

```javascript
if (e.type === 'mouseout' ||
    e.x < area.left || e.x > area.right ||
    e.y < area.top || e.y > area.bottom) {
    me._active = []
} else {
    me._active = me._chart.getElementsAtEventForMode(...
}
```

Reply:
Good idea I forgot about accounting for y. I'll make the changes, thanks!